### PR TITLE
alreadyVisited cookie ranamed to the gu.alreadyVisited

### PR DESCRIPTION
**Pull request by Calanthe**

### CODE REVIEW

**Comment by davidrapson** [comment]
Not actually a cookie I hope :smile: 

:+1: With the caveat that this is going to reset anything using the value of the counter, so assuming we're OK with that?

**Comment by johnduffell** [comment]
but why? it's already in our domain

**Comment by davidrapson** [comment]
All other localStorage items that we control are prefixed `gu.` so this is mostly just for consistency I imagine. Plus it helps to distinguish between localStorage items set by third-parties.

File: common/app/templates/headerInlineJS/bootSystemJS.scala.js [hunk 1]
```diff
@@ -95,8 +95,8 @@ System['import']('core').then(function () {
                     images.listen();
 
                     if (guardian.isModernBrowser) {
-                        alreadyVisted = storage.local.get('alreadyVisited') || 0;
-                        storage.local.set('alreadyVisited', alreadyVisted + 1);
+                        alreadyVisted = storage.local.get('gu.alreadyVisited') || 0;
```
Comment (jbreckmckye) [inline common/app/templates/headerInlineJS/bootSystemJS.scala.js]: Should we use the name `gu.already.visited`, in the convention set by the likes of `gu.prefs.messages`, `gu.history.popular`, et cetera?

Comment (rich-nguyen) [inline common/app/templates/headerInlineJS/bootSystemJS.scala.js]: happy with `gu.alreadyVisited`, it's fairly sensible and there is no convention upheld at the moment.

Comment (jbreckmckye) [inline common/app/templates/headerInlineJS/bootSystemJS.scala.js]: @rich-nguyen Fair enough!

**Pull request merged**